src/components/misc/StaticMap.jsx

```diff
@@ -17,39 +19,66 @@ export default class StaticMap extends React.Component {
     };
 
     render() {
-        let loc = this.props.location;
-        let lat = loc.lat;
-        let lng = loc.lng;
-
-        const imgSrc = url.format({
-            protocol: 'https',
-            hostname: 'maps.googleapis.com',
-            pathname: '/maps/api/staticmap',
-            query: {
-                center: (lat + 0.0002) + ',' + lng,
-                markers: lat + ',' + lng,
-                zoom: 17,
-                size: this.props.width + 'x' + this.props.height,
-                key: 'AIzaSyAHVagqI3RTd0psf57oA6gzKqVyjp8FS8w',
-            }
-        });
-
-        // Convert lat/lng to degrees, minutes and seconds, which is more
-        // readable than the decimal value with tons of decimals.
-        let latDMS = convertLatToDMS(lat), lngDMS = convertLngToDMS(lng);
-        let latStr = latDMS.deg.toString()
-            .concat('° ', latDMS.min, '\' ', latDMS.sec, '" ', latDMS.dir);
-        let lngStr = lngDMS.deg.toString()
-            .concat('° ', lngDMS.min, '\' ', lngDMS.sec, '" ', lngDMS.dir);
+        let placeholder = this.props.placeholder;
+        let map = null;
+        let coordinates = null;
 
-        return (
-            <div className="StaticMap"
-                onClick={ this.onClick.bind(this) }>
+        if (placeholder) {
+            map = (
+                 <div className="StaticMap-placeholder">
+                    <Msg id="panes.addLocation.setPositionLink"/>
```

Comment:
This should either be configurable from the outside (via a property) or the string should be moved from `panes.addLocation` to `misc.staticMap` since `StaticMap` is a component that should be self-contained, regardless of where it's used in the future.

In fact, it is already used in two different panes so coupling it's internal strings to `AddLocationPane` is already not ideal.

Personally I prefer just moving the strings to `misc.staticMap` over adding properties. It's easier, and I don't see a case when being able to change the strings would be useful.